src/main/antora/modules/ROOT/pages/elasticsearch/template.adoc

```diff
@@ -213,7 +213,7 @@ Using `StringQuery` may be appropriate if you already have an Elasticsearch quer
 `NativeQuery` is the class to use when you have a complex query, or a query that cannot be expressed by using the `Criteria` API, for example when building queries and using aggregates.
 It allows to use all the different `co.elastic.clients.elasticsearch._types.query_dsl.Query` implementations from the Elasticsearch library therefore named "native".
 
-The following code shows how to search for persons with a given firstname and for the found documents have a terms aggregation that counts the number of occurences of the lastnames for these persons:
+The following code shows how to search for persons with a given `firstname` and for the found documents have a terms aggregation that counts the number of occurrences of the `lastnames` for these persons:
 
```

Comment:
when marking it this way we are referring to an actual field/property; in alignment with the following could that should then be `firstName` and `lastName` and in the code the "`last-name`" should be fixed as well.

And while you are fixing that file, line 238 should be
```
[[elasticsearch.operations.searchtemplatequery]]
```